features/FxA-56-email-deliverability/README.md

```diff
@@ -1,66 +1,160 @@
+**we can take a data-driven approach to improving email deliverability**
+of the authentication system.
 
-If we stop sending additional email messages to bounced emails addresses then our "complaint rate" with popular email providers will decrease and improve our deliverability.
+If we avoid sending additional emails to addresses that are known to bounce then
+**our "sender reputation" wth popular email providers will improve**
+and hence our email deliverability will improve.
 
 
 ## Metrics
 
-The following metrics should help us understand and debug email deliverability:
+The following metrics will be used
+to understand and debug email deliverability:
 
-* Flow events for email bounces
-* Current bounce rate
-* A dashboard that lists common
+### Completion Rate for Email Flows
 
-----
+For flows that result in an email being sent,
+we will produce a flow funnel graph
+showing the rate of dropoff for the following events:
 
-## Detailed design
+* The email being successfully delivered by SES
+* The link in the email being clicked
+* The flow completing
+
+The graph will look something like this:
+
+![Mock graph for completion rate of email flows](email_flow_completion_rate.jpg)
+
+This will help us to know what percentage of flows
+fail to due email delivery issues.
+
+Future attempts to improve email deliverability
+can be judged successful
+if they improve this completion rate.
+
+### Lost Mail Rate for Email Flows
+
+For flows that result in an email being sent,
+we will produce a graph showing the percentage for which:
 
-There are different parts that will help us solve our email deliverability problems:
+* We receive a successful delivery notification from SES
+* We receive a bounce notification from SES
+* we receive no further information about delivery
 
-* Adding flow events for bounces
+The graph will look something like this:
 
-The plan is to add flow event metrics to our existing bounce tracking scripts in the `fxa-auth-server`. This approach will help us create a funnel metrics graph.
+![Mock graph for lost mail rate of email flows](lost_mail_rate.jpg)
 
-* Creating a "complaint rate" and a "bounce rate" dashboard
+This will help us to know whether deliverability issues
+are caused by issues downstream, upstream, or inside of SES.
 
-This dashboard would include some of the details we get
-from Amazon Web Services.
+### Bounce and complaint rate by template
 
-We can break down the dashboards by the email template and / or language where possible.
+We will produce a graph showing the number of bounce notifications per day,
+as a percentage of the number of emails sent that day,
+broken down by email template name.
+It will look something like this:
 
-Additionally, we can keep a record in our database, creating a table recording bounced emails. We can evolve what we do with records in the database as we learn.
+![Mock graph for bounce mail by template](bounce_rate_by_template.jpg)
```

Comment:
Looks great!
Any way we can track locale on this? Each template has a lot of different variations. Since we don't have full control of localization, our volunteers might accidentally add a few of spam triggers into the copy. (examples of English spam triggers: free, click here, $$$, etc) 

Just saying, it'll be hard to come to any conclusion about spam rate for a template if we can't dig in per language.